lib_eio_linux/eio_linux.ml

```diff
@@ -871,6 +927,18 @@ let net = object
     let sock = FD.of_unix ~sw ~seekable:false ~close_unix:true sock_unix in
     Low_level.connect sock addr;
     (flow sock :> <Eio.Flow.two_way; Eio.Flow.close>)
+
+  method datagram_socket ~sw = function
+    | `Udp (host, port) ->
+      let host = Eio_unix.Ipaddr.to_unix host in
+      let addr = Unix.ADDR_INET (host, port) in
+      let sock_unix = Unix.socket Unix.PF_INET Unix.SOCK_DGRAM 0 in
+      (* TODO: Should expose these as arguments. *)
```

Comment:
Do you think this should be exposed in the function type like for listening connections?

Reply:
I'm happy to keep them internal until someone complains.